geomesa-feature/src/main/scala/org/locationtech/geomesa/feature/SimpleFeatureEncoder.scala

```diff
@@ -57,16 +58,17 @@ trait SimpleFeatureDecoder extends HasEncoding {
 }
 
 object SimpleFeatureDecoder {
+
   def apply(sft: SimpleFeatureType, encoding: FeatureEncoding) =
     encoding match {
+      case FeatureEncoding.KRYO => new KryoFeatureEncoder(sft, sft)
       case FeatureEncoding.AVRO => new AvroFeatureDecoder(sft)
       case FeatureEncoding.TEXT => new TextFeatureDecoder(sft)
```

Comment:
(idly) I wonder if there's a Scala compiler check to make sure that all Values in an Enumeration/Value class/whatnot are handled in a match statement.

I'm thinking along the lines of a sealed trait...